<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryDelineationIts4landWindow</class>
 <widget class="QDialog" name="BoundaryDelineationIts4landWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Its4land Load</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="loginGroupBox">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="loginInput">
        <property name="placeholderText">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsPasswordLineEdit" name="passwordInput">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="toolTip">
      <string>Connect to its4land platform</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="projectsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Projects</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="horizontalLayout_3">
      <item row="1" column="0">
       <widget class="QListWidget" name="projectsListWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Projects on its4land platform</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="projectsErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Unable to load project from its4land platform</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color : red;</string>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="projectsDetailsGroupBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Details about its4land project</string>
        </property>
        <property name="title">
         <string>Project details</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QFormLayout" name="projectsFormLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="projectsNameLabel">
             <property name="toolTip">
              <string>Project name</string>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="projectsNameValueLabel">
             <property name="toolTip">
              <string>Project name</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="projectsDescriptionLabel">
             <property name="toolTip">
              <string>Project description</string>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="projectsDescriptionValueLabel">
             <property name="toolTip">
              <string>Project description</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="projectsTagsLabel">
             <property name="toolTip">
              <string>Project tags</string>
             </property>
             <property name="text">
              <string>Tags</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="projectsTagsValueLabel">
             <property name="toolTip">
              <string>Project tags</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="projectsModelsLabel">
             <property name="toolTip">
              <string>Project models</string>
             </property>
             <property name="text">
              <string>Models</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="projectsModelsValueLabel">
             <property name="toolTip">
              <string>Project models</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="projectsSpatialSourcesLabel">
             <property name="toolTip">
              <string>Project spatial source</string>
             </property>
             <property name="text">
              <string>Spatial Sources</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="projectsSpatialSourcesValueLabel">
             <property name="toolTip">
              <string>Project spatial source</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="projectsBaseLayersComboBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="projectsLoadBaseLayerButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Load base layer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select base layer:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="validationSetsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Image segmentation lines</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Image segmentation lines</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="horizontalLayout">
      <item row="1" column="0">
       <widget class="QListWidget" name="validationSetsListWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Image segmentation lines on its4land platform</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="validationSetsErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Unable to load image segmentation lines from its4land platform</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color : red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="validationSetsDetailsGroupBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Details about its4land image segmenation lines</string>
        </property>
        <property name="title">
         <string>Image segmentation details</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QFormLayout" name="validationSetsformLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="validationSetsNameLabel">
             <property name="toolTip">
              <string>Segments layer name</string>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="validationSetsNameValueLabel">
             <property name="toolTip">
              <string>Segments layer name</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="validationSetsDescriptionLabel">
             <property name="toolTip">
              <string>Segments layer description</string>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="validationSetsDescriptionValueLabel">
             <property name="toolTip">
              <string>Segments layer tags</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="validationSetsTagsLabel">
             <property name="toolTip">
              <string>Segments layer tags</string>
             </property>
             <property name="text">
              <string>Tags</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="validationSetsTagsValueLabel">
             <property name="toolTip">
              <string>Segments layer tags</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="validationSetsModelsLabel">
             <property name="toolTip">
              <string>Segments layer models</string>
             </property>
             <property name="text">
              <string>Models</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="validationSetsSizeLabel">
             <property name="toolTip">
              <string>Segments layer feature count</string>
             </property>
             <property name="text">
              <string>Feature count</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="validationSetsModelsValueLabel">
             <property name="toolTip">
              <string>Segments layer models</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="validationSetsSizeValueLabel">
             <property name="toolTip">
              <string>Segments layer feature count</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="validationSetsLoadButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Load segments layer from its4land platform</string>
           </property>
           <property name="text">
            <string>Load segments layer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boundaryStringsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Boundary lines</string>
     </property>
     <property name="title">
      <string>Boundary lines</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="boundaryStringsErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Unable to load boundary lines from its4land platform</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color : red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="boundaryStringsformLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="boundaryStringsNameLabel">
          <property name="toolTip">
           <string>Boundary layer name</string>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="boundaryStringsNameValueLabel">
          <property name="toolTip">
           <string>Boundary layer name</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="boundaryStringsFeaturesLabel">
          <property name="toolTip">
           <string>Boundary layer features</string>
          </property>
          <property name="text">
           <string>Features</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="boundaryStringsFeaturesValueLabel">
          <property name="toolTip">
           <string>Boundary layer features</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="boundaryStringsSizeLabel">
          <property name="toolTip">
           <string>Boundary layer feature count</string>
          </property>
          <property name="text">
           <string>Feature count</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="boundaryStringsSizeValueLabel">
          <property name="toolTip">
           <string>Boundary layer feature count</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="boundaryStringsLoadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Load boundary layer from its4land platform</string>
        </property>
        <property name="text">
         <string>Load layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boundaryStringsUploadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Upload boundary layer to its4land platform</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Upload boundary layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>buttonBox</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BoundaryDelineationIts4landWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BoundaryDelineationIts4landWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
